kiwi/config/functions.sh

```diff
@@ -25,6 +25,36 @@ export LC_ALL=C
 #======================================
 # Common Functions
 #--------------------------------------
+function mountCgroups {
+    mount -t tmpfs cgroup_root /sys/fs/cgroup
+    for group in devices blkio cpuset pids; do
+        mkdir -p /sys/fs/cgroup/"${group}"
+        mount -t cgroup "${group}" -o "${group}" /sys/fs/cgroup/"${group}"
+    done
+}
```

Comment:
I trust you on all these functions, I never done something like this, so I have no experience. Looks good without knowing the details.

Reply:
This is just a simple cgroup setup in a new tmpfs. I found out that podman cannot be used with cgroups disabled, even though there is a configuration option for it available it complains at "podman run", as there is no systemd in an obs env I choose to use cgroupfs and a new cgroup's mount.

I did not add this information to the method as it just creates a new cgroup mount list which is a common task

Do you think we need more information ?

Reply:
Did you try with OBS' systemd-nspawn mode for builds instead of using simple chroot?

Reply:
I was not aware that you can choose this in any way ? are there docs ? And even if it can be selected I have the requirement that it has to work from within a chroot.

Reply:
Yes, it's selectable in sysconfig files.

Reply:
Maybe you can give me a pointer into that and how it hooks into obs, on the matrix channel ? Thanks much

Other than that if nobody objects I would go forward with the implementation for now ? The way it's provided in this PR I have tested and works fine in obs as well as in the private instance at EB for which it relates to the job

Thanks